Packs/CommonPlaybooks/Playbooks/playbook-Get_File_Sample_From_Path_-_Generic_V3.yml

```diff
@@ -217,7 +224,248 @@ tasks:
+      playbookName: Cortex XDR - Retrieve File v2
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      endpoint_ids:
+        complex:
+          root: inputs.Agent_ID
+          transformers:
+          - operator: uniq
+      file_path:
+        complex:
+          root: inputs.Path
+          transformers:
+          - operator: uniq
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1310,
+          "y": 340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: 093a8aa7-07c9-45e3-85de-b08c39983dce
+    type: playbook
+    task:
+      id: 093a8aa7-07c9-45e3-85de-b08c39983dce
+      version: -1
+      name: CrowdStrike Falcon - Retrieve File
+      description: |-
+        This playbook is part of the 'Malware Investigation And Response' pack. For more information, refer to https://xsoar.pan.dev/docs/reference/packs/malware-investigation-and-response.
+        This playbook retrieves and unzips files from CrowdStrike Falcon and returns a list of the files that were and were not retrieved.
+      playbookName: CrowdStrike Falcon - Retrieve File
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      HostId:
+        complex:
+          root: inputs.Agent_ID
+          transformers:
+          - operator: uniq
+      PathsToGet:
+        complex:
+          root: inputs.Path
+          transformers:
+          - operator: uniq
+      ZipPassword:
+        simple: infected
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": -360,
+          "y": 340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: 8232fa6d-94a4-4d02-8ea4-dd0d3144fcaa
+    type: playbook
+    task:
+      id: 8232fa6d-94a4-4d02-8ea4-dd0d3144fcaa
+      version: -1
+      name: MDE - Retrieve File
+      description: |-
+        This playbook is part of the 'Malware Investigation And Response' pack. For more information, refer to https://xsoar.pan.dev/docs/reference/packs/malware-investigation-and-response.
+        This playbook uses the Live Response feature to retrieve a file from an endpoint./nNote that the endpoint id will be set from the incident field "DeviceID".
+      playbookName: MDE - Retrieve File
```

Comment:
This Playbook was intended to be only for a dedicated use case. The thing here is that the retrieve tasks require endpoint id -> which was set to incident.Deviceid . So I assume you can create another playbook that fits you parent's PB logic, or just remove this one - up to you.